src/spec/src/browser.ts

```diff
@@ -73,7 +76,10 @@ export const getOptions = (
         ...options['git-hosts'],
       }
     : defaultGitHosts,
-  registries: options?.registries ?? {},
+  registries: {
+    ...defaultRegistries,
+    ...(options?.registries ?? {}),
+  },
```

Comment:
Let's revert this change:

```suggestion
  registries: options?.registries ?? {},
```